plugins/it.xsemantics.dsl/src/it/xsemantics/dsl/XsemanticsModelGenerator.xtend

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
```

Comment:
But why is it required? The JvmModelInferrer already handles this automatically... why am I forced to add the comments myself? This is a breaking changes that break all existing use cases of this feature... (even the documentation is out of date https://github.com/LorenzoBettini/xsemantics/blob/master/doc/it.xsemantics.doc/xdoc/XsemanticsSyntax.xdoc).

If this is required only to avoid the empty space I think one could customize the part in Xbase that handles that.